src/org/traccar/protocol/MeitrackProtocolDecoder.java

```diff
@@ -155,9 +155,14 @@ private Position decodeRegularMessage(Channel channel, SocketAddress remoteAddre
                 position.set(Position.PREFIX_ADC + i, parser.nextHexInt(0));
             }
         }
-
-        position.set(Position.KEY_BATTERY, parser.nextHexInt(0));
-        position.set(Position.KEY_POWER, parser.nextHexInt(0));
+        
+        if (Context.getConfig().hasKey(getProtocolName() + ".devicetype") == "tc68s") {
```

Comment:
We have a special model field in the device model. I think we should use that instead.